golos.publication/golos.publication.cpp

```diff
@@ -465,16 +469,15 @@ void publication::close_message(structures::mssgid message_id) {
     for(auto& ben : mssg_itr->beneficiaries) {
         auto ben_payout = int_cast(elai_t(payout) * ELF(ben.deductprcnt));
         eosio_assert((0 <= ben_payout) && (ben_payout <= payout - ben_payout_sum), "LOGIC ERROR! publication::payrewards: wrong ben_payout value");
-        payto(ben.account, eosio::asset(ben_payout, state.funds.symbol), static_cast<enum_t>(payment_t::VESTING));
+        payto(ben.account, eosio::asset(ben_payout, state.funds.symbol), static_cast<enum_t>(payment_t::VESTING), memo_message_id + ", type of payment: benefeciary");
```

Comment:
I suggest form memo as: `{author|curator|benefeciary} reward for post <author>:<permlink>:<ref_block_num>`. It's easy to read for users, and parse by scripts.